data/python

```diff
@@ -4,3 +4,4 @@ pypi.io
 pypi.org
 python.org
 pythonhosted.org
+piwheels.org
```

Comment:
> piwheels is a community project by Ben Nuttall and Dave Jones.

I think this domain should be included in another category, or at least add a comment to separate the official projects and the community projects.

Reply:
what do you think we add a new file, maybe named as `python-community`?

Reply:
> what do you think we add a new file, maybe named as `python-community`?

I'm OK with that ;-)

Reply:
Done